<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>12</width>
    <height>12</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Mode_Lb">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>420</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>MODE: Local</string>
    </property>
    <property name="buddy">
     <cstring>Next_btn</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_filename">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>280</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>The last character_number cannot be changed.</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Text that matches audio, if provided</string>
    </property>
    <property name="text">
     <string>Transcripton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Play1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Play</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../icons/icons.qrc">
      <normaloff>:/play.png</normaloff>:/play.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Stop1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Stop</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/stop.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Record1">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Record</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/record.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Play2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Play</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/play.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Stop2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>280</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Stop</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/stop.png</normalon>
     </iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="Record2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>280</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Record</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/record.png</normalon>
     </iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>User 1 Audio (Target)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>280</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>User 2: File Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Next_btn">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>410</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Will save current recording if recorded. Update filenames, load next clip, next transcript line.</string>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="Transcript_brw">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>561</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="WAV_Graph2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>551</width>
      <height>81</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Wave form to record. Try to match the wave form above.</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="Wav_Graph1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>551</width>
      <height>81</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Wave form of first audio snip.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>410</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Decrements line number, filename, and load previous.</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>31</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_filename_target">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>160</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>57</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Waiting</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_filename_end">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>280</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>The last character_number cannot be changed.</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionAudio_Settings"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionManual"/>
   </widget>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAudio_Settings">
   <property name="text">
    <string>Audio Settings</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionTutorial">
   <property name="text">
    <string>Tutorial</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="text">
    <string>Manual</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>lineEdit_filename</tabstop>
  <tabstop>Play1</tabstop>
  <tabstop>Stop1</tabstop>
  <tabstop>Record1</tabstop>
  <tabstop>Play2</tabstop>
  <tabstop>Stop2</tabstop>
  <tabstop>Record2</tabstop>
  <tabstop>Next_btn</tabstop>
  <tabstop>Transcript_brw</tabstop>
  <tabstop>WAV_Graph2</tabstop>
  <tabstop>Wav_Graph1</tabstop>
 </tabstops>
 <resources>
  <include location="../icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
